.changelog/22972.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/aws_ami_copy: Prevent panic when reading the source AMI's `boot_mode` or `destination` `outpost_arn`
+```
```

Comment:
```suggestion
```release-note:bug
resource/aws_ami_copy: Prevent panic when reading the source AMI's `boot_mode` or `destination` `outpost_arn`
```

```release-note:bug
resource/aws_ami_from_instance: Prevent panic when reading the source AMI's `boot_mode` or `destination` `outpost_arn`
```
```